charts/jenkins/values.yaml

```
@@ -123,6 +123,10 @@ controller:
   # Share process namespace to allow sidecar containers to interact with processes in other containers in the same pod
   shareProcessNamespace: false
 
+  # Service links might cause issue if running in a namespace with a large amount of services
+  # that might cause a slow startup when plugins are copied from ref to volume
+  enableServiceLinks: true
```

I can't see how this would impact Jenkins, setting this to false should help everyone?

see also https://github.com/kubernetes/kubernetes/issues/121787
```suggestion
  enableServiceLinks: false
```

---

Fine more me. I just didn't wanted to create "breaking" changes

---

This is fixed and rebased